### ISSUE TITLE
又发现一处bug

### ISSUE THREAD

**Issue by tencentdosos** [Plachtaa/VITS-fast-fine-tuning#109]
![image](https://user-images.githubusercontent.com/127338628/226000799-c6f9501e-58e7-4480-b6e9-deeb981fd55b.png)

中文不知道为啥会变成这样...

**Comment by Plachtaa** [comment]
看看你的.json文件

**Comment by tencentdosos** [comment]
> 看看你的.json文件

{
  "train": {
    "log_interval": 10,
    "eval_interval": 100,
    "seed": 1234,
    "epochs": 10000,
    "learning_rate": 0.0002,
    "betas": [
      0.8,
      0.99
    ],
    "eps": 1e-09,
    "batch_size": 16,
    "fp16_run": true,
    "lr_decay": 0.999875,
    "segment_size": 8192,
    "init_lr_ratio": 1,
    "warmup_epochs": 0,
    "c_mel": 45,
    "c_kl": 1.0
  },
  "data": {
    "training_files": "final_annotation_train.txt",
    "validation_files": "final_annotation_val.txt",
    "text_cleaners": [
      "chinese_cleaners"
    ],
    "max_wav_value": 32768.0,
    "sampling_rate": 22050,
    "filter_length": 1024,
    "hop_length": 256,
    "win_length": 1024,
    "n_mel_channels": 80,
    "mel_fmin": 0.0,
    "mel_fmax": null,
    "add_blank": true,
    "n_speakers": 1,
    "cleaned_text": true
  },
  "model": {
    "inter_channels": 192,
    "hidden_channels": 192,
    "filter_channels": 768,
    "n_heads": 2,
    "n_layers": 6,
    "kernel_size": 3,
    "p_dropout": 0.1,
    "resblock": "1",
    "resblock_kernel_sizes": [
      3,
      7,
      11
    ],
    "resblock_dilation_sizes": [
      [
        1,
        3,
        5
      ],
      [
        1,
        3,
        5
      ],
      [
        1,
        3,
        5
      ]
    ],
    "upsample_rates": [
      8,
      8,
      2,
      2
    ],
    "upsample_initial_channel": 512,
    "upsample_kernel_sizes": [
      16,
      16,
      4,
      4
    ],
    "n_layers_q": 3,
    "use_spectral_norm": false,
    "gin_channels": 256
  },
  "symbols": [
    "_",
    "\uff1b",
    "\uff1a",
    "\uff0c",
    "\u3002",
    "\uff01",
    "\uff1f",
    "-",
    "\u201c",
    "\u201d",
    "\u300a",
    "\u300b",
    "\u3001",
    "\uff08",
    "\uff09",
    "\u2026",
    "\u2014",
    " ",
    "A",
    "B",
    "C",
    "D",
    "E",
    "F",
    "G",
    "H",
    "I",
    "J",
    "K",
    "L",
    "M",
    "N",
    "O",
    "P",
    "Q",
    "R",
    "S",
    "T",
    "U",
    "V",
    "W",
    "X",
    "Y",
    "Z",
    "a",
    "b",
    "c",
    "d",
    "e",
    "f",
    "g",
    "h",
    "i",
    "j",
    "k",
    "l",
    "m",
    "n",
    "o",
    "p",
    "q",
    "r",
    "s",
    "t",
    "u",
    "v",
    "w",
    "x",
    "y",
    "z",
    "1",
    "2",
    "3",
    "4",
    "5",
    "0",
    "\uff22",
    "\uff30"
  ],
  "speakers": {
    "adao": 0
  }
}

**Comment by tencentdosos** [comment]
在训练完的测试里没问题的.转到本机之后就这样了..

**Comment by Plachtaa** [comment]
你的cleaner名字为什么是`chinese_cleaners`

**Comment by tencentdosos** [comment]
> 你的cleaner名字为什么是`chinese_cleaners`

23333,因为我只开启了中文...我看有注释,可以只训练中文..但是出来还是一股子大佐味...

**Comment by tencentdosos** [comment]
我去掉了,这个参数.生产出来是电音..主要问题是,在云端的时候.他就是很正常....没有一点问题..

**Comment by Plachtaa** [comment]
呃呃，你自己非要用我有什么办法，本来就没有写好，那个模型本身效果也不行
